native/cocos/editor-support/spine-wasm/AtlasAttachmentLoaderExtension.h

```diff
@@ -5,11 +5,11 @@
 
 class AttachmentVertices {
 public:
-    AttachmentVertices(int verticesCount, uint16_t *triangles, int trianglesCount, uint32_t textureId);
+    AttachmentVertices(int verticesCount, uint16_t *triangles, int trianglesCount, const spine::String& textureId);
```

Comment:
In this PR, you changed the parameter type from `uint32_t` to `const spine::String &` which indicates the `region-page-name`, it's a texture atlas name, right? If so , using the old `textureId` name seems not to be suitable.

Reply:
After call the function updateAttachmentVerticesTextureId, texture will store texute's uuid

Reply:
So I recommend to rename the parameter name to `textureUUID`.

Reply:
Fixed